src/main/java/org/skriptlang/skript/bukkit/InputKey.java

```diff
@@ -0,0 +1,50 @@
+package org.skriptlang.skript.bukkit;
+
+import org.bukkit.Input;
+
+import java.util.EnumSet;
+import java.util.Set;
+
+public enum InputKey {
+
+	FORWARD("forward movement key"),
+	BACKWARD("backward movement key"),
+	RIGHT("right movement key"),
+	LEFT("left movement key"),
+	JUMP("jumping key"),
+	SNEAK("sneaking key"),
+	SPRINT("sprinting key");
+
+	private final String toString;
+
+	InputKey(String toString) {
```

Comment:
The lang values should probably be used instead rather than hardcoding it here. Maybe just take in the language node and create a Noun for each?